core/domain/stats_domain.py

```diff
@@ -117,30 +117,28 @@ class StateStats(object):
     """Domain object representing analytics data for an exploration's state."""
```

Comment:
Just to check, does this domain object pertain to a specific exploration version as well? If so, might want to state that in the docstring.

Reply:
Done.